src/Client.php

```diff
@@ -161,29 +194,60 @@ private function post($route = '', array $body = [])
     }
 
     /**
-     * Perform cURL DELETE request to route. Used by
+     * Perform cURL GET request to route. For example used by
      * Alerts API requests: https://www.stathat.com/manual/alerts_api
      *
-     * @param string $route
-     * Ex: /x/ACCESSTOKEN/alerts/ALERTID
+     * @param string $curl_url
+     *   Ex: https://www.stathat.com/x/ACCESSTOKEN/alerts
+     * @return array or string $result
+     *   An array of alert details or the response code string in the case of a non 200 response.
      */
-    private function deleteAlertCurl($route = '')
+    private function curlGet($curl_url)
```

Comment:
I'd be happy to work through a refactor with you using #8. My plan is to first get all the basic functionality flushed out  first. Sorry if it's stating to feel *icky*.